### PR TITLE
[cherry-pick] Disable starting of monitoring reconciler

### PR DESCRIPTION
**Pull request by VedantMahabaleshwarkar**
Cherry Pick #162

### CODE REVIEW

File: config/base/params.env [hunk 1]
```diff
@@ -1,2 +1 @@
-monitoring-namespace=opendatahub
 odh-model-controller=quay.io/opendatahub/odh-model-controller:v0.11.1.1
```
Comment (spolti) [inline config/base/params.env:1]: shouldnt be v0.11.1.2?

Comment (terrytangyuan) [inline config/base/params.env:1]: This is different from the original PR https://github.com/opendatahub-io/odh-model-controller/pull/162

**Comment by spolti** [comment]
do we still need this PR?

**Comment by israel-hdez** [comment]
> do we still need this PR?

Echoing...

**Comment by VedantMahabaleshwarkar** [comment]
PR not needed, this change is already in main and will land in 0.12 when that branch is cut from main

**Pull request closed without merging**